docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,790 @@
+    ")\n",
+    "text2 = (\n",
+    "    \"All 3 wave modes are plotted for Hollweg's relation (solid lines) and Bellan's solut- \\n\"\n",
+    "    \"ion (dashed lines) with $\\\\beta = 2.0$ and $\\\\theta \\\\approx 90^{\\\\circ}$.\"\n",
+    ")\n",
+    "plt.figtext(-0.08, -0.18, text2, ha=\"left\", transform=ax.transAxes, fontsize=15.5)\n",
+    "ax.text(0.57, 0.95, text1, transform=ax.transAxes, fontsize=18)\n",
+    "ax.legend(handles=[p4, p1, p5, p2, p6, p3], fontsize=14, ncol=3, loc=\"upper left\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Hollweg 1999 and Bellan 2012 comparison\n",
+    "\n",
+    "[bellan2012]: https://doi.org/10.1029/2012JA017856\n",
+    "\n",
+    "Figure 1 of [Bellan 2012][bellan2012] chooses parameters such that\n",
+    "$\\beta = 0.4$ and $\\Lambda = 0.4$. Below we define parameters to\n",
+    "approximate Bellan's assumptions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define input parameters\n",
+    "inputs = {\n",
+    "    \"B\": 400e-4 * u.T,\n",
+    "    \"ion\": Particle(\"He+\"),\n",
+    "    \"n_i\": 6.358e19 * u.m ** -3,\n",
+    "    \"T_e\": 20 * u.eV,\n",
+    "    \"T_i\": 10 * u.eV,\n",
+    "    \"theta\": np.linspace(0, 90) * u.deg,\n",
+    "    \"k\": (2 * np.pi * u.rad) / (0.56547 * u.m),\n",
+    "}\n",
+    "\n",
+    "# a few useful plasma parameters\n",
+    "params = {\n",
+    "    \"n_e\": inputs[\"n_i\"] * abs(inputs[\"ion\"].charge_number),\n",
+    "    \"cs\": cs_(inputs[\"T_e\"], inputs[\"T_i\"], inputs[\"ion\"]),\n",
+    "    \"wci\": wc_(inputs[\"B\"], inputs[\"ion\"]),\n",
+    "    \"va\": va_(inputs[\"B\"], inputs[\"n_i\"], ion=inputs[\"ion\"]),\n",
+    "}\n",
+    "params[\"beta\"] = (params[\"cs\"] / params[\"va\"]).value ** 2\n",
+    "params[\"wpe\"] = wp_(params[\"n_e\"], \"e-\")\n",
+    "params[\"Lambda\"] = (inputs[\"k\"] * params[\"va\"] / params[\"wci\"]).value ** 2\n",
+    "\n",
+    "(params[\"beta\"], params[\"Lambda\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# compute omegas for Bellan\n",
+    "bellan_omegas = two_fluid(**inputs)\n",
+    "\n",
+    "# compute omegas for Hollweg\n",
+    "fast = []\n",
+    "alfven = []\n",
+    "acoustic = []\n",
+    "for val in inputs[\"theta\"]:\n",
+    "    hollweg_inputs = {\n",
+    "        **inputs,\n",
+    "        \"theta\": val,\n",
+    "    }\n",
+    "    fast.append(hollweg(**hollweg_inputs)[\"fast_mode\"].value)\n",
+    "    alfven.append(hollweg(**hollweg_inputs)[\"alfven_mode\"].value)\n",
+    "    acoustic.append(hollweg(**hollweg_inputs)[\"acoustic_mode\"].value)\n",
+    "hollweg_omegas = {\n",
+    "    \"fast_mode\": fast * u.rad / u.s,\n",
+    "    \"alfven_mode\": alfven * u.rad / u.s,\n",
+    "    \"acoustic_mode\": acoustic * u.rad / u.s,\n",
+    "}"
```

Comment:
Since #1529 has been merged, we should make use of the ability of defining arrays for both `k` and `theta`.